deploy/kubernetes/seqr/seqr.gcloud.yaml

```diff
@@ -127,6 +127,11 @@ spec:
             secretKeyRef:
               name: seqr-secrets
               key: google_client_secret
+        - name: SERVICE_ACCOUNT_FOR_ANVIL
+          valueFrom:
+            secretKeyRef:
+              name: seqr-secrets
+              key: service_account_id
```

Comment:
I think these codes are shared by both `dev` and `prod` env. Do they?

Reply:
yeah that seems easiest

Reply:
but can you rename this secret to `anvil_service_account`, as I can imagine in the future we might have multiple different service accounts and this could be confusing